### ISSUE TITLE
test-aggregateColor fails without farver package

### ISSUE THREAD

**Issue by brownag** [ncss-tech/aqp#110]
@dylanbeaudette 

This test needs to check whetehr farver 2.0.2 is installed and compare results accordingly? Or choose a less sensitive test case?

```
test-aggregateColor.R:56: failure: manual calculation, single profile
`test` not equal to "7.5YR 3/2".
1/1 mismatches
x[1]: "10YR 3/2"
y[1]: "7.5YR 3/2"
```

Tests fine after installing latest version of farver off CRAN (which I did not have yet)

**Comment by dylanbeaudette** [comment]
Thanks for catching that. I'll fix.

**Comment by brownag** [comment]
Fix c3eab86
